output/output.py

```diff
@@ -0,0 +1,8 @@
+from output.status import Status
+
+class OutPut:
+    def __init__(self, result = None, message = None, status = None):
```

Comment:
You aren't set the result and status in the instance variable, so why are you passing them ??!!! you need to have 

pass an exception here. 
message = str(exception) if excetion not one else "success"
self.status = success if not exception else exception.status